### PR TITLE
Add example config for octo

### PR DESCRIPTION
**Pull request by Uzaaft**
Add example config for octo.nvim, taken from https://github.com/LunarVim/LunarVim/pull/2040

### CODE REVIEW

**Comment by Uzaaft** [comment]
Works in my config. This should effectively enable us to close the previously mentioned issue.

File: docs/plugins/02-extra-plugins.md [hunk 1]
```diff
@@ -228,6 +228,177 @@ Every plugin that works with Neovim works with LunarVim, here are some examples
   event = "BufRead",
```
Comment (kylo252) [inline docs/plugins/02-extra-plugins.md:228]: The `BufRead`  isn't particularly useful for lazy-loading, instead maybe use `cmd = "Octo"`

Also, don't you still need to call the setup function?

File: docs/plugins/02-extra-plugins.md [hunk 2]
```diff
@@ -228,6 +228,177 @@ Every plugin that works with Neovim works with LunarVim, here are some examples
...
+   return M._config
+ end
+
+ function M.config()
+   lvim.builtin.octo = {
+     active = false,
+     on_config_done = nil,
+     opts = M.defaults,
+   }
+ end
```
Comment (kylo252) [inline docs/plugins/02-extra-plugins.md:312]:
```suggestion
```

This is probably irrelevant for user-plugins

File: docs/plugins/02-extra-plugins.md [hunk 3]
```diff
@@ -228,6 +228,177 @@ Every plugin that works with Neovim works with LunarVim, here are some examples
...
+   require("octo.completion")
+   require("octo.folds")
+   _G.octo_repo_issues = {}
+   _G.octo_buffers = {}
+   require("octo.signs").setup()
+   user_config = user_config or {}
+   M._config = require("octo.utils").tbl_deep_clone(M.defaults)
+   require("octo.utils").tbl_soft_extend(M._config, user_config)
+
+   M._config.file_panel = vim.tbl_deep_extend("force", M.defaults.file_panel, user_config.file_panel or {})
```
Comment (kylo252) [inline docs/plugins/02-extra-plugins.md:324]: I think you missed moving have these to `get_config()` 😄

**Pull request closed without merging**